### ISSUE TITLE
A few comments

### ISSUE THREAD

**Issue by SteOtt** [ransome1/sleek#37]
Hello,
I am using sleek for a few days now and I like your software really a lot.
Despite its 0.xx version code I'm impressed by the stability and the features it has.
However I have a few remarks but all of them are minor:
- The shortcut Strg+b (to show/hide filter pane) is missing in the Shortcuts help 
![grafik](https://user-images.githubusercontent.com/17461591/106953079-41820d80-6732-11eb-8b43-d1f27a4f7204.png)
- A shortcut to disable the filters is missing
- Could you shorten the links in the tasks to improve readability
- It would be nice, if you could add some other schemes to the URI-detection like e.g. file:// which would allow to link files
- The tabindex could be improved sometimes. E.g. if you are in the filter line a tab doesn't bring you to the task pane
- Generaly I would like to navigate through the tasks and edit them with my keyboard
- Could you support hidden tasks (h:1) as other todo.txt apps do?
- And the best would be, if you could support notes in tasks as the notes add on for todo.txt-cli (https://github.com/mgarrido/todo.txt-cli/tree/note/todo.actions.d) does.

Thanks for your good work and keep on with sleek.

BR
Stefan

**Comment by ransome1** [comment]
@SteOtt Thank you for your feedback, it's highly appreciated. All your points are valid and while I will work on the minor ones soon, please let me add the note function to my backlog. It would be an interesting addition, but up until v1.0 I committed myself to only work on technical depths. If the v1.0 milestone is reached, its' features again.

**Comment by ransome1** [comment]
```
The shortcut Strg+b (to show/hide filter pane) is missing in the Shortcuts help
A shortcut to disable the filters is missing
Could you shorten the links in the tasks to improve readability
```
These three will be included in the 0.9.0-Beta

Also the tabindex improvements will be part of the v0.9.x. The other points are material for 1.0 and beyond.

**Comment by ransome1** [comment]
I adapted your suggestions to my backlog and hope you're fine with me closing this issue.

**Comment by ransome1** [comment]
Hi @SteOtt , the hidden todos feature has been implemented in v0.9.8 and needs testing: https://github.com/ransome1/sleek/issues/76#issuecomment-808787906
